src/routes/Home/FrontLayer/Home/Home.frontLayer.js

```diff
@@ -16,9 +16,41 @@ const useStyles = makeStyles(styles)
 const HomeFrontLayer = () => {
   const classes = useStyles()
   const { t } = useTranslation('homePage')
-  const [state] = useSharedState()
+  const [bpsData, setBpsData] = useState([])
+
+  useEffect(() => {
+    const loadBps = async () => {
+      const bpNames = await novotebuyeosUtil.getBps()
+      const bps = (
+        await Promise.resolve(
+          bpNames.reduce(async (previous, current) => {
```

Comment:
@tetogomez please, let me know if there is a better way to handle a reduce function with an async call inside, or well, if you could realize a better approach to do the dame a reduce function does.

Reply:
nice!